reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -53,7 +53,7 @@ import org.ossreviewtoolkit.reporter.utils.ReportTableModelMapper
 import org.ossreviewtoolkit.reporter.utils.SCOPE_EXCLUDE_LIST_COMPARATOR
 import org.ossreviewtoolkit.reporter.utils.containsUnresolved
 import org.ossreviewtoolkit.utils.ORT_FULL_NAME
-import org.ossreviewtoolkit.utils.isValidUrl
+import org.ossreviewtoolkit.utils.isValidUri
```

Comment:
commit message: "[..] URI is sufficient [..]" -> "[..] URI this is sufficient [..]"?

Reply:
Actually, I believe the sentence to be correct. More verbosely I wanted to say "using an URI (instead of an URL) is sufficient".